plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -45,6 +44,7 @@ import {
   CatalogFilterItem,
 } from '../CatalogFilter/CatalogFilter';
 import { CatalogTable } from '../CatalogTable/CatalogTable';
+import useStaleWhileRevalidate from 'swr';
```

Comment:
Not useSWR? :grin:

Reply:
When I saw the hook for the first time, I had no idea what was going on. So I thought I'd try to call it something a little more semantic. haha. 

https://en.wikipedia.org/wiki/SWR

Reply:
Yeah true, although tbh I don't think StaleWhileRevalidate is too obvious either x)

Reply:
`useReallyGoodCachyLibraryThing`

Reply:
useRGCLT

Reply:
That's what I'm talking about.

Reply:
Nailed it